src/app/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+  Button,
+  EmptyState as EmptyStatePF,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Title,
+} from "@patternfly/react-core";
+import { PlusCircleIcon } from "@patternfly/react-icons";
+import { Trans } from "@rhoas/app-services-ui-components";
+
+interface EmptyStateProps {
+  createButton: {
+    title: string;
+    onCreate: () => void;
+    isDisabled?: boolean;
+  };
+  quickStartGuide?: {
+    i18nKey: string;
+    onQuickstartGuide: () => void;
+  };
+  title: string;
+}
+
+export const EmptyState = ({
+  createButton,
+  quickStartGuide,
+  title,
+}: EmptyStateProps): JSX.Element => {
+  return (
+    <EmptyStatePF variant={EmptyStateVariant.xs}>
```

Comment:
Using the `small` variant will add a bit of extra padding around the empty state. It looks too close to the parent now.

```suggestion
    <EmptyStatePF variant={EmptyStateVariant.small}>
```